pyorientAssignment.py

```diff
@@ -35,15 +35,31 @@
 # To find the minimum price, create a variable and initialize it to a very large number, 
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
-
+'''
 for record in records:
 	print record.price
+'''
+minPrice=500
+maxPrice=20000
+sumPrice=0
+
+for record in records:
+    sumPrice+=record.price
+    if record.price<minPrice:
+        minPrice=minPrice
+    else:
```

Comment:
You do not need the else statement there. If the price is not smaller, you just go to the next check.